### PR TITLE
Handling the minimal tls version for MySQL and Postgres

### PR DESCRIPTION
**Pull request by akamath46297**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S163.

**Comment by Juliehzl** [comment]
Hi @akamath46297, can you add test for you new parameter and make CI pass?

**Comment by yonzhan** [comment]
@akamath46297, any update for this PR?

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -884,6 +884,10 @@ Release History
 * storage blob generate-sas: User delegation SAS token support with --as-user
 * storage container generate-sas: User delegation SAS token support with --as-user
 
+**RDBMS**
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: Please make the release note at the right position, which should be the top of the file.

**Comment by yonzhan** [comment]
@akamath46297, any update for this PR?

**Comment by yonzhan** [comment]
@akamath46297, any update for this PR?

**Comment by yonzhan** [comment]
@akamath46297, any update for this PR?

**Comment by akamath46297** [comment]
@yonzhan awaiting MS deployment to get the tests out.

**Comment by Juliehzl** [comment]
> @yonzhan awaiting MS deployment to get the tests out.

Given not ready in deployment, I will mark the PR as DoNotMerge now. When it is ready, please let me know and let's start the merge process.

**Comment by qwordy** [comment]
There are 1119 files changed...
It seems you mess up the commit history. Try `git reset`, commit you changes, `git push -f`

**Comment by bim-msft** [comment]
Hi @akamath46297 please `rebase` dev branch to your code in your local environment and then `git push -f`.

**Pull request closed without merging**